source/javascripts/application.js

```diff
@@ -39,8 +39,8 @@
         $('.js-menu-section').each(function() {
             var $section = '#' + $(this).attr('id');
             var $sectionLink = $('.nav-list a[href$="' + $section + '"]');
-
             if ($($section).detectSection()) {
+                $section === '#work' ? window.history.pushState({}, document.title, '/') : null;
```

Comment:
This replaces the #work in the URL when we're in that section. Ideally, this scroll handler would be more efficient in general.